Localization/TellMeWhen-enUS.lua

```diff
@@ -1915,6 +1917,8 @@ L["DT_DOC_ValueMax"] = "Returns the maximum of the numerical value that the icon
 L["DT_INSERTGUID_TOOLTIP"] = "|cff7fffffShift-click|r to insert this icon's identifier into a DogTag."
 L["DT_INSERTGUID_GENERIC_DESC"] = [[If you would like one icon to display information about another, |cff7fffffShift-click|r that icon to insert its unique identifier that you can pass as the tag's "icon" parameter.]]
 
+L["DT_KEYSTONE_DESC"] = "Level of the currently active mythic keystone"
```

Comment:
Redundant with `CONDITIONPANEL_KEYSTONE_DESC` - don't need two of the same string. I have no qualms with using a string keyed as `CONDITIONPANEL` as a description for a DogTag. The localization keys are pretty much a free-for-all (unfortunately).